src/app/product-list/product-list.component.css

```diff
@@ -0,0 +1,8 @@
+.strike-through {
+    text-decoration: line-through;
+}
+
+.my-7 {
+    margin-top: 14px;
```

Comment:
Actually, please don't use margin, rather use padding on this component

Reply:
This class is specific to fixed margin spacing in the y-axis